### PR TITLE
feat: added new function for getting transfers by contract address

### PR DESCRIPTION
**Pull request by javiergarciavera**

### CODE REVIEW

File: integration-test/sdk.nfts.transfers.test-integration.ts [hunk 1]
```diff
@@ -366,4 +369,49 @@ describe('E2E Test: Sdk (read)', () => {
       );
     });
   });
+
+  describe('As an account I should get list of transfers by a contract address', () => {
+    const contractAddress = '0xbc4ca0eda7647a8ab7c2061c2e118a18a936f13d'; //BAYC
+    jest.setTimeout(10000);
```
Comment (sahar-fehri) [inline integration-test/sdk.nfts.transfers.test-integration.ts:375]: why did you need to put the setTimeout here?

**Review by sahar-fehri — APPROVED** [review]

File: integration-test/sdk.nfts.transfers.test-integration.ts (hunk 1 above)
Comment (javiergarciavera) [reply integration-test/sdk.nfts.transfers.test-integration.ts:375]: because sometimes I notice that Moralis takes quite a lot to respond, so I do not want the test to fail if it's Moralis related and it returns data after all

File: package.json [hunk 2]
```diff
@@ -69,6 +69,7 @@
     "typescript": "^4.7.4"
   },
   "dependencies": {
+    "@typescript-eslint/parser": "^5.49.0",
```
Comment (salimtb) [inline package.json:72]: why did you add this package ?

**Review by salimtb — APPROVED** [review]
